tests/helper.ts

```diff
@@ -43,3 +47,30 @@ export function wait(time: number): Promise<void> {
         );
     });
 }
+
+export async function tryUpsert(client: dgraph.DgraphClient, query: string, mutation: dgraph.Mutation, blankNodeLabel: string)
+: Promise<void> {
+    const txn = client.newTxn();
+
+    const req = new dgraph.Request();
+    req.setQuery(query);
+    req.setMutationsList([mutation]);
+    req.setCommitNow(true);
+
+    let uid: string;
+    try {
+        // Update account only if matching uid found.
+        const response = await txn.doRequest(req);
+        uid = response.getUidsMap().get(blankNodeLabel);
+        expect(uid).not.toEqual("");
+    } finally {
+        await txn.discard();
+    }
+}
+
+export const QUERY_PERMISSION_DENIED = new Error("7 PERMISSION_DENIED:\
```

Comment:
These error constants shouldn't be in the shared helper file, as they are test specific (predicate names are hardcoded) and at least the first one is used only once